src/App.js

```diff
@@ -0,0 +1,20 @@
+import React, { useEffect } from 'react';
+import { BrowserRouter, Route, Routes } from 'react-router-dom';
+import BikeList from './features/bikeList/BikeList';
+import './App.css';
+import store from './app/store';
+import getBikeList from './features/bikeList/bikeListAPI';
+
+export default function App() {
+  useEffect(() => {
+    store.dispatch(getBikeList());
+  });
+
+  return (
+    <BrowserRouter>
+      <Routes>
+        <Route path="/" element={<BikeList />} />
+      </Routes>
+    </BrowserRouter>
```

Comment:
- [ ] You suppose to have at least two routes here one for the `home` page and the other for the `details` page, so Please add the other route it will be something like that:
```javascript
<Routes>
        <Route path="/" element={<Home />} />
        <Route path="/:country_id" element={<Details/>} />
 </Routes>
```

Reply:
Hi! @Omar-Muhamad both issues were solved.